coops-ui/src/constants/index.ts

```diff
@@ -1,4 +1,5 @@
 import * as DirectorConst from './director-constants'
+export * from './address-constants'
```

Comment:
100% Agree, i will refactor this and import/export properly. I noticed this and was looking for a good place to do the refactor, thanks for commenting on it!

Reply:
So i refactored these a little different than mixins. To avoid having to refer to the named `as` export where we are using these constants i did an `export * `. It cleans up the code a little and also simplified the constants file. Refer to the changes on SummaryDirectors to see an example.

Reply:
I'm going to do the same Refactor to the interfaces, as it makes the code much cleaner when implemented. I should have done it this way to start :)

Reply:
Looks good.